3rdParty/velocypack/src/Dumper.cpp

```diff
@@ -376,7 +376,7 @@ void Dumper::dumpValue(Slice const* slice, Slice const* base) {
     }
 
     case ValueType::Object: {
-      ObjectIterator it(*slice);
+      ObjectIterator it(*slice, true);
```

Comment:
I would prefer making this a flag in `velocypack::Options`, which can set to "dump in any order" by default in ArangoDB.